src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6982,5 +6982,19 @@ public override IOperation VisitUsingDeclaration(IUsingDeclarationOperation oper
             throw ExceptionUtilities.Unreachable;
         }
 
+        public override IOperation VisitWithExpression(IWithExpressionOperation operation, int? argument)
+        {
+            EvalStackFrame frame = PushStackFrame();
+            // Initializer is removed from the tree and turned into a series of statements that assign to the cloned instance
+            IOperation visitedInstance = Visit(operation.Value);
+
+            IOperation cloned = operation.CloneMethod is null
+                ? MakeInvalidOperation(visitedInstance.Type, visitedInstance)
+                : new InvocationOperation(operation.CloneMethod, visitedInstance,
+                    isVirtual: false, arguments: ImmutableArray<IArgumentOperation>.Empty,
```

Comment:
>false [](start = 31, length = 5)

Based on [https://github.com/dotnet/roslyn/pull/44852](https://github.com/dotnet/roslyn/pull/44852), the call probably should be virtual

Reply:
Thanks. Will fix in a follow-up PR

Reply:
@AlekseyTs It looks like Andy fixed this already (we now have `isVirtual: true` in `master`.